src/acceptance-test/java/org/zalando/nakadi/webservice/hila/StreamBatch.java

```diff
@@ -36,30 +40,54 @@ public SubscriptionCursor getCursor() {
         return unmodifiableList(events);
     }
 
+    public StreamMetadata getMetadata() {
```

Comment:
I think no. I think we should think from the perspective of the user who uses the class. When he calls `getMetadata` he should be aware that he can get null. (the fact that it is Nullable in constructor will not really help in this situation)